src/Pages/Product/Product.js

```diff
@@ -0,0 +1,247 @@
+import React, { useState } from 'react';
+import { GlobalStyle } from '../../Styles/GlobalStyle';
+import { theme } from '../../Styles/theme';
+import styled from 'styled-components';
+
+function Product() {
+  // componentDidMount() {
+  //   fetch('http://localhost:3000/data/dummy.json', {
+  //     method: 'GET'
+  //   })
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       this.setState({
+  //         commentList: data,
+  //       });
+  //     });
+  // }
+
+  return (
+    <ProductInner>
+      <GlobalStyle />
+      <Container>
+        <Header>
+          <Detail>
+            <li>
+              <P>원룸</P>
+              <div>
+                <H3>6000만원</H3>
+              </div>
+            </li>
+            <Li1>
+              <P>전용면적</P>
+              <H3>19.2평</H3>
+            </Li1>
+            <Li2>
+              <P>위코드부동산</P>
+              <Call>연락처보기</Call>
+            </Li2>
+          </Detail>
+          <DetailInner></DetailInner>
+        </Header>
+        <Info>
+          <Confirm>
+            <ConfirmDetail1>매물 2021</ConfirmDetail1>
+            <ConfirmDetail2>확인한 매물입니다.</ConfirmDetail2>
+          </Confirm>
+          <InfoDetail>
+            <List>
+              <Tittle>해당층/건물층</Tittle>
+              <ListDetail>3층</ListDetail>
+            </List>
+          </InfoDetail>
+        </Info>
+        <div>
+          <Photo>
+            <Number>매물 번호 20730964</Number>
+            <PhotoInner></PhotoInner>
+          </Photo>
+        </div>
+        <div>
+          <TextDetail>
+            <TextDetail1>
+              서초역 도보 6분 풀옵션 원룸 위치 좋고 가격 좋은 신축급 컨디션
+              깔끔한 집
+            </TextDetail1>
+            <TextDetail2>
+              -조용한 주택가에 위치한 다가구 원룸 -입구는 1층이나 창문쪽은
+              지층이 있어 2층 높이 -새로 도배를 했으며 기본적인 수리는 마쳐놓은
+              상태 -샤시는 이번에 새로 함 -인근에 이 동네 번화가가 위치해 있어
+              생활이 편리 -롯데슈퍼가 1분거리에 있고 서초역 부근에 롯데마트가
+              있음 -조금 멀리 보자면 서울 성모병원이나 고속터미널 남부터미널
+              백화점등이 버스로 3~4정거장 내에 있어 이용이 편리 -반려동물은
+              불가x
+            </TextDetail2>
+          </TextDetail>
+        </div>
+      </Container>
+    </ProductInner>
+  );
+}
+
+//스타일 컴포넌트
+const ProductInner = styled.div`
+  width: 100%;
+`;
+
+const Container = styled.div`
+  width: 1200px;
+  height: 100%;
+  margin: 0px auto;
+  padding: 0px 10em;
+`;
+
+const Header = styled.div`
+  align-items: center;
+  width: 100%;
+  padding-top: 35px;
+  margin-bottom: 25px;
+`;
+
+const Detail = styled.ul`
+  margin-bottom: 35px;
+  display: flex;
+`;
+
+const DetailInner = styled.div`
+  display: flex;
+  height: 22px;
+`;
+
+const P = styled.p`
+  margin-bottom: 5px;
+  font-size: 1.2rem;
+  color: ${props => props.theme.gray};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.gray};
```
함수 파라미터 자리에서 직접 구조분해 할당이 가능합니다!

Reply:
감사합니다!